kubernetes/services/ignite_alcor_ip.yaml

```diff
@@ -0,0 +1,131 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: ignite-alcor-ip
+roleRef:
+  kind: ClusterRole
+  name: ignite-alcor-ip
+  apiGroup: rbac.authorization.k8s.io
+subjects:
+  - kind: ServiceAccount
+    name: ignite-alcor-ip
+    namespace: ignite-alcor-ip
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: ignite-alcor-ip
+  namespace: ignite-alcor-ip
+rules:
+  - apiGroups:
+      - ""
+    resources: # Here are resources you can access
+      - pods
+      - endpoints
+    verbs: # That is what you can do with them
+      - get
+      - list
+      - watch
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ignite-alcor-ip
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ignite-alcor-ip
+  namespace: ignite-alcor-ip
+---
+apiVersion: v1
+kind: Service
+metadata:
+  # The name must be equal to TcpDiscoveryKubernetesIpFinder.serviceName
+  name: ignite-alcor-ip-service
+  # The name must be equal to TcpDiscoveryKubernetesIpFinder.namespaceName
+  namespace: ignite-alcor-ip
+  labels:
+    app: ignite-alcor-ip
+spec:
+  #type: LoadBalancer
+  ports:
+    - name: rest
+      port: 8080
+      targetPort: 8080
+    - name: sql
+      port: 10801
+      targetPort: 10800
+    #- name: thinclients
+      #port: 10900
+      #targetPort: 10900
+  selector:
+    app: ignite-alcor-ip
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  labels:
+    app: ignite-alcor-ip
+  name: ignite-alcor-ip
+  namespace: ignite-alcor-ip
+spec:
+  selector:
+    matchLabels:
+      app: ignite-alcor-ip
+  serviceName: ignite-alcor-ip-service
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: ignite-alcor-ip
+    spec:
+      serviceAccountName: ignite-alcor-ip
+      containers:
+        - name: ignite-alcor-ip-node
+          image: ignite_alcor:lib8
+          imagePullPolicy: IfNotPresent
+          env:
+            - name: OPTION_LIBS
+              value: ignite-kubernetes,ignite-rest-http
+            - name: CONFIG_URI
+              value: https://raw.githubusercontent.com/kevin-zhonghao/alcor/xml/kubernetes/services/ignite_ip_config.xml
+            #- name: IGNITE_QUIET
+              #value: "false"
+            #- name: JVM_OPTS
+              #value: "-Djava.net.preferIPv4Stack=true \
```

Comment:
@kevin-zhonghao Do we need to turn on this JVM option?

Reply:
Yes ~ I turned on this option already